test/test_latex_renderer.py

```diff
@@ -51,12 +51,20 @@ def test_image(self):
         self._test_token('Image', output, src='src')
 
     def test_link(self):
-        output = '\\href{target}{inner}'
-        self._test_token('Link', output, target='target')
+        for target, output in {
```

Comment:
Dare to rewrite this to a parameterized test? :) Search for `@parameterized.expand` in the existing code - I've started to use that just recently...

Reply:
Done in 4231e80d

Reply:
Great, thank you.